src/frontend/komponenter/Behandling/VedtakOgBeregning/VedtaksresultatSwitch.tsx

```diff
@@ -30,6 +60,40 @@ const VedtaksresultatSwitch: React.FC<Props> = (props: Props) => {
     const [inntektBegrunnelse, settInntektBegrunnelse] = useState<string>('');
     const [laster, settLaster] = useState<boolean>(false);
     const { vedtaksresultatType, behandlingId, settFeilmelding } = props;
+    const tomVedtaksperiodeRad = {
+        periodeType: '' as EPeriodetype,
+        aktivitet: '' as EAktivitet,
+        datoFra: '',
+        datoTil: '',
+    };
+    const [vedtaksperiodeListe, settVedtaksperiodeListe] = useState<IPeriode[]>([
+        tomVedtaksperiodeRad,
```

Comment:
Dette som initial state gjør at innsending uten å sette noen perioder feiler. Usikker på om det skal være mulig, men det bør uansett løses på en bedre måte.

Reply:
Hva skjer dersom vi har tom liste som initiell state? Da må saksbehandler trykke på knappen for å legge til en rad? "Greit nok" inntil videre?

Reply:
Innsending av tom liste er lov, men usikker på om det bør være det. Så denne løsningen funker jo som en "sperre" midlertidig. Men tror uansett vi bør legge til "slett periode"-knapp før vi kan merge, og da blir ting kanskje litt annerledes.